opencga-app/app/scripts/azure/arm/appgw/keyvault.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "location": {
+        "type": "string",
+        "defaultValue": "[resourceGroup().location]",
+        "metadata": {
+          "description": "Specifies the Azure location where the key vault should be created."
+        }
+      },
+      "tenantId": {
+        "type": "string",
+        "defaultValue": "[subscription().tenantId]",
+        "metadata": {
+          "description": "Specifies the Azure Active Directory tenant ID that should be used for authenticating requests to the key vault. Get it by using Get-AzSubscription cmdlet."
```

Comment:
Powershell specific comment. Might be better to reference the `az` command here, given that it's more likely the user will be on a nix box.

Reply:
Removed PS command in 
https://github.com/opencb/opencga/pull/1202/commits/438ce991d65fbb83ec0c1bc399c020be7eaa9b46.

Suggestion: maybe remove the tenantId from the params everywhere? And just make it a variable that points to `[subscription().tenantId]` since it's not possible to deploy these resources over multiple subscriptions anyway.. 

Thoughts?

Reply:
Changed to variable in https://github.com/opencb/opencga/pull/1202/commits/331c1f0b566e2a562b95040e0af8be4a26d9bf12